src/components/input.js

```diff
@@ -175,15 +175,15 @@ export class AutocompleteTextInput extends Component {
   }
 
   render() {
-    const { value, onChange, suggestions, ...props } = this.props
+    const { value, onChange, suggestions, renderSuggestion = _.identity, theme, ...props } = this.props
     const { show } = this.state
     return h(Autosuggest, {
       id: this.id,
       inputProps: { value, onChange: e => onChange(e.target.value) },
       suggestions: show ? (value ? _.filter(Utils.textMatch(value), suggestions) : suggestions) : [],
       onSuggestionsFetchRequested: () => this.setState({ show: true }),
       onSuggestionsClearRequested: () => this.setState({ show: false }),
-      onSuggestionSelected: (e, { suggestionValue }) => onChange(suggestionValue),
+      onSuggestionSelected: (e, { suggestionValue }) => onChange(suggestionValue, true),
```

Comment:
This component was designed to handle a very basic case, a plain text input with autocomplete, but what we want for this feature is substantially different. Rather than adding all these overrides here, it might be cleaner to implement a new 'picker' component, built with Autosuggest, that has the exact set of behaviors we want.